src/Project/BuilderCfgFile.ts

```diff
@@ -78,6 +82,32 @@ export class BuilderCfgFile extends EventEmitter implements helpers.FileSelector
     this.logger.outputLine('Add Import: ' + inputModel);
   }
 
+  private cfgOptimize(prop: any) {
+    let optimize = new JobOptimize();
+    optimize.inputPath = prop[K_INPUT_PATH];
+    optimize.outputPath = prop[K_OUTPUT_PATH];
+
+    let inputModel = path.basename(optimize.inputPath);
+    optimize.name = 'Optimize ' + inputModel;
+
+    console.log('optimize = ', optimize);
+    this.jobOwner.addJob(optimize);
+    this.logger.outputLine('Add Optimize: ' + inputModel);
+  }
+
+  private cfgQuantize(prop: any) {
+    let quantize = new JobQuantize();
+    quantize.inputPath = prop[K_INPUT_PATH];
+    quantize.outputPath = prop[K_OUTPUT_PATH];
+
```

Comment:
@seanshpark Other options will be added?

Reply:
Yup, shortly after current draft all lands :)